### DISCUSSION TITLE
Welcome to pyscreenrec Discussions!

### DISCUSSION THREAD

**Discussion by shravanasati** [shravanasati/pyscreenrec#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
hi looks cool ı learn python can you make document please
1) video output is so fast
2) press ı try but press key not working (ı learn python maybe my fault this)
3) sound record coming soon?

sorry my english not good

**Reply by ghost** [reply]
https://youtu.be/dV1KGoD-7_o
